Bindings/Java/Matlab/Utilities/prescribeMotionInModel.m

```diff
@@ -71,6 +45,7 @@ function prescribeMotionInModel(Model_In, Mot_In, Model_Out)
 
 % Initialize model
 osimModel=Model(modelfilepath);
+osimModel.finalizeFromProperties();
```

Comment:
as of #1748 you don't need to call `finalizeFromProperties()` explicitly after loading a model from file.